index.html

```diff
@@ -5067,6 +5123,7 @@ <h3>The <dfn>JsonLdProcessor</dfn> Interface</h3>
             <code>application/ld+json</code>,
             nor any other media type using a
             <code>+json</code> suffix as defined in [[RFC6839]],
+            <span class="changed">or if the processor is a <a>pure JSON Processor</a></span>,
```

Comment:
The whole sentence is a bit complex to parse... But it seems to me that this last part ("or if the processor...") wrongly implies that a pure JSON Processor will reject any media-type.
More precisely, I read the whole sentence as IF (media-type neither X,Y...) OR (pure JSON processor) THEN reject.

I suggest rewriting it as:
* If content-type is text/html and the processor is a pure JSON Processor, reject.
* Otherwise, if the content-type is neither application/ld+json..., reject.

Reply:
Yes, I was being lazy in not trying to rewrite the text more, but it was indeed, incorrect.